pypika/dialects.py

```diff
@@ -79,6 +88,23 @@ def modifier(self, value):
         """
         self._modifiers.append(value)
 
+    @builder
+    def load(self, fp):
+        self._load_file = fp
```

Comment:
does it make sense to have this on the query instance? There isn't a corresponding test case for it. Do you intend for load to be the first function called to create the query and for into to always follow, or is there some case where a query might be built using other functions first then load is called?

Reply:
We can limit it to be being the first function called for now and into having to follow. I'll update it